docs/RELEASE.md

```diff
@@ -1,3 +1,30 @@
+# Release 0.4 - 12/6/2018
+
+## Major Features
+  * [Kubeflow Training service](./KubeflowMode.md)
+    * Support tf-operator
+    * [Distributed trial example](../examples/trials/mnist-distributed/dist_mnist.py) on Kubeflow
+  * [Grid search tuner](../src/sdk/pynni/nni/README.md#Grid) 
+  * [Hyperband tuner](../src/sdk/pynni/nni/README.md#Hyperband)
+  * Support launch NNI experiment on MAC
+  * WebUI 
+    *  UI support for hyperband tuner
+    *  Remove tensorboard button 
+    *  Show experiment error message 
+    *  Show line numbers in search space and trial profile
+    *  Support search a specific trial by trial number
+    *  Show trial's hdfsLogPath
+    *  Download experiment parameters
```

Comment:
"Download experiment results" would be better.

Reply:
Yes

Reply:
Done